packages/mui-joy/src/styles/ColorSystem.ts

```diff
@@ -15,38 +44,48 @@ export interface PaletteRange {
   700: string;
   800: string;
   900: string;
-  channel500: string;
 }
 
 export interface PaletteText {
-  heading: React.CSSProperties['color'];
-  headingIntro: React.CSSProperties['color'];
-  content: React.CSSProperties['color'];
-  detail: React.CSSProperties['color'];
-  overline: React.CSSProperties['color'];
+  primary: React.CSSProperties['color'];
+  secondary: React.CSSProperties['color'];
+  tertiary: React.CSSProperties['color'];
 }
 
-export interface PaletteBgNeutral {
-  transparency: React.CSSProperties['backgroundColor'];
-  plain: React.CSSProperties['backgroundColor'];
+export interface PaletteBackground {
+  default: React.CSSProperties['backgroundColor'];
+  level1: React.CSSProperties['backgroundColor'];
+  level2: React.CSSProperties['backgroundColor'];
+  level3: React.CSSProperties['backgroundColor'];
 }
 
 export interface ColorPalettePropOverrides {}
 
+export type DefaultColorPalette =
+  | 'primary'
+  | 'neutral'
+  | 'danger'
+  | 'info'
+  | 'success'
+  | 'warning'
+  | 'context';
```

Comment:
What's `context`?

Reply:
Components can change the color to `context` to enable the contextual override feature. This is added after chatting with @oliviertassinari because it makes the variables easier to reason. More detail https://www.notion.so/mui-org/RFC-Theme-implementation-e15bc2ef11ba4f6f804434af27141afe#cdcb3ea92fcc44d798d7de6a77971537